portfolio/src/main/webapp/script.js

```diff
@@ -17,12 +17,35 @@
  */
 function addRandomGreeting() {
   const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
+      ['Hello!', '¡Hola!', 'Ni Hao!', 'Bonjour!', 'Namaste!', 
+      'Sawasdee Kha!', 'Konnichiwa!'];
 
   // Pick a random greeting.
   const greeting = greetings[Math.floor(Math.random() * greetings.length)];
 
   // Add it to the page.
   const greetingContainer = document.getElementById('greeting-container');
+
+  const curr = greetingContainer.innerText
+  if (curr === greeting) {
+    greeting = greetings[Math.floor(Math.random() * greetings.length)];
```

Comment:
greeting cannot be `const` since you are re-assigning it here, it should be a `let`. I'm surprised your type checker let this though ;)